core/app/modules/kit-library/assets/js/pages/index/index-no-results.js

```diff
@@ -1,8 +1,41 @@
 import { Heading, Text, Grid, Button } from '@elementor/app-ui';
+import { useMemo } from 'react';
+import { useNavigate, useLocation } from '@reach/router';
 
 import './index-no-results.scss';
 
+function useNoResultContent( isFilterActive, clearFilter ) {
+	const navigate = useNavigate();
+	const location = useLocation();
+
+	return useMemo( () => {
+		const content = [
+			{
+				title: __( 'No favorites here yet...', 'elementor' ),
+				description: __( 'Use the heart icon to save kits that inspire you. You\'ll be able to find them here', 'elementor' ),
+				CTAText: __( 'Continue Browsing', 'elementor' ),
+				CTAAction: () => navigate( '/kit-library' ),
```

Comment:
Maybe `ctaAction` / `ctaText`?
Since those are the only text & action, is that even necessary to use "CTA" in the variable?

Reply:
Changed to lower case. 
and yes we should recognize that this content is for the "call to action" button and not for the content of the page.